Conductor/Client/Constants.cs

```diff
@@ -83,5 +83,9 @@ public static class Constants
         //Annotation
         public const string TOKENCANCELLATION = "Token Requested Cancel";
         public const string RUNTIMEERROR = "Method failed with runtime error";
+
+        //Environment Variable
```

Comment:
prob for testing, we should remove it from here.

Reply:
> prob for testing, we should remove it from here.

@v1r3n  updated the PR.